.gitignore

```diff
@@ -4,3 +4,4 @@ dist/
 *.kra
 *.kra~
 *.png~
+.idea
```

Comment:
Maybe we could use something more granular? https://stackoverflow.com/questions/11968531/what-to-gitignore-from-the-idea-folder

I'm using VSCode now but there might be other IntelliJ-ers

Reply:
I used to try this approach for a few years, but once every 2 months they come up with a new garbage file (and a new backwards-incompatible IDE version), so this is just the easy solution. Some files like run configuration might be a good idea to keep, but it's always a discussion about what we want to share and keep personalizaed for each user. So I'd keep everything ignored until we find something that we know we want to share

Reply:
Makes sense!